bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -283,5 +337,5 @@ spec:
         resources:
           requests:
             storage: {{ .Values.persistence.size | quote }}
-        {{ include "cassandra.storageClass" . }}
-{{- end }}
+        {{ include "common.storage.class" (dict "persistence" .Values.persistence "global" .Values.global) }}
```

Comment:
I'd add `-` and `nindent` so we have more control of where that piece of yaml should be. Otherwise it could be more prone to errors